src/sparseml/transformers/finetune/data/data_args.py

```diff
@@ -62,6 +62,14 @@ class DataTrainingArguments:
         default=512,
         metadata={"help": "Number of samples to use for one-shot calibration"},
     )
+    dvc_dataset_path: Optional[str] = field(
+        default=None,
+        metadata={"help": "Path to dvc dataset to load, of format dvc://path"},
+    )
+    dvc_data_registry: Optional[str] = field(
```

Comment:
Could we perhaps make those two arguments more clear for the user? I can imagine most of the users, not being familiar with dvc, may need additional information. To my understanding, registry in dvc is the repository that can be accessed at `dvc_dataset_path` right? But it is a bit weird to see both 'registry' and 'repository' being mentioned here.